File: rafael/controle-de-fluxo/ContaUns.java
class ContaUns {
	public static void main(String[] args) {
		double numero = Math.random();
		System.out.println(numero);

		String s = "" + numero;
		int resposta = 0;		

		for(int i = 0; i < s.length(); i++) {
			if(s.charAt(i) == '1') {	
				resposta++;
			}
		}

		System.out.println(resposta);
	}
}

File: rafael/controle-de-fluxo/ContaUns.cs
class ContaUns
{
	static void Main()
	{
		System.Random gerador = new System.Random();

		double numero = gerador.NextDouble();
		System.Console.WriteLine(numero);

		string s = "" + numero;
		int resposta = 0;		

		for(int i = 0; i < s.Length; i++)
		{
			if(s[i] == '1')
			{	
				resposta++;
			}
		}

		System.Console.WriteLine(resposta);
	}
}
